src/Menu/Builder/AdminContent.php

```diff
@@ -49,20 +49,40 @@ public function build(MenuEntry $root)
                 ->setPermission('dashboard')
         );
 
-        // ContentTypes, where show_in_menu is set true
-        $contentTypes = $this->contentTypes->filter(function ($k, $v) {
-            return $v['show_in_menu'] === true;
-        });
-        foreach ($contentTypes as $name => $contentType) {
-            $this->addContentType($mainContentRoot, $name, $contentType);
+        foreach ($this->contentTypes as $name => $contentType) {
+            if ($contentType['show_in_menu'] === true) {
```

Comment:
`$this->contentTypes` is a recursive `Bag`, ergo `$contentType` is a `Bag`, use the getter please.

Reply:
Also, you're looking for the exception to the logical flow, it would be cleaner to flip the test, i.e. `!== true`